articles/protocols/saml/adfs.md

```diff
@@ -4,202 +4,139 @@ toc: true
-
-::: note
-The sign in and sign out URLs are usually in the form of `https://your.adfs.server/adfs/ls`.
-:::
-
-Click **Save**.
-
-## Step 3: Edit the Relying Party Trust
-
-Once you have set the required parameters for creating your Auth0 connection and clicked **Save**, you'll see a page with instructions on creating a new relying party trust in ADFS. You'll need the following parameters:
-
-| Parameter | Sample Value |
-| - | - |
-| Post-back URL | `https://{yourAuth0accountdomain}/login/callback}/login/callback?connection={your new SAML connection}` |
-| Entity ID | `urn:auth0:account:connection` |
-
-The set of instructions presented to you after you've created your new connection will also have the exact values required for your Auth0 account/connection.
+### Export the Signing Certificate
 
-In the ADFS console, go to **ADFS > Relying Party Trusts** using the left-hand navigation pane. Select the relying party trust you created in step 1 and click **Properties** (located on the right-hand navigation pane). 
+Finally, you'll need to export the signing certificate from the ADFS console to upload it to Auth0.
 
-![](/media/articles/protocols/saml-adfs/saml22.png)
+1. Using the left-hand navigation pane, go to **ADFS > Service > Certificates**. Select the **Token-signing** certificate, and right click to select **View Certificate**.
+2. On the **Details** tab, click **Copy to File...**. This launches the **Certificate Export Wizard**. Click **Next**.
+3. Choose **Base-64 encoded X.509 (.CER)** as the format you'd like to use. Click **Next**.
+4. Provide the location to where you want the certificate exported. Click **Next**.
+5. Verify that the settings for your certificate are correct and  click **Finish**.
 
-Switch over to the **Identifiers** tab, and populate the **Relying Party Identifier** with the **Entity ID** value. Be sure to click **Add** to add the identifier to your list.
+## Create a SAML connection with Auth0 as the service provider
 
-![](/media/articles/protocols/saml-adfs/saml23.png)
+1. Follow the tutorial on creating a SAML connection where [Auth0 acts as the service provider](/protocols/saml/saml-sp-generic). Where prompted, upload the signing certificate you exported from ADFS.
 
-Switch over to the **Endpoints** tab, and select the placeholder URL you provided earlier. Click **Edit...**.
+    The sign in and sign out URLs are usually in the form of `https://your.adfs.server/adfs/ls`.
 
-![](/media/articles/protocols/saml-adfs/saml24.png)
+2. Click **Save**. A page with instructions for creating a new Relying Party Trust in ADFS appears displaying the exact values required for your Auth0 account/connection. Make a note of these values. 
 
-Populate the **Trusted URL** with the **Post-back URL** value.
+    Here is an example: 
+    
+    | Parameter | Example Value |
+    | - | - |
+    | Post-back URL | `https://{yourAuth0accountdomain}/login/callback}/login/callback?connection={your new SAML connection}` |
```

Comment:
I'd be good to explain either here or below in Trusted URL step of "Edit the Relying Party Trust" that {yourAuth0accountdomain} refers to the custom domain is one is being used.

Reply:
Added note